### PR TITLE
[RPA-868] Fix how actions display for users

### PR DESCRIPTION
**Pull request by MRSIMM0**
## Description

Admin:
image

User:
image

Guest:
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

---

I agree to the terms of the RunBotics Contributor License Agreement.

### CODE REVIEW

File: runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionListPanel/ActionList/ActionList.tsx [hunk 1]
```diff
@@ -115,15 +115,16 @@ const ActionList: FC<ActionListProps> = ({
...
-                                                    item.id
-                                                );
+                                                !hasAdvancedActionsAccess &&
+                                                (isGroupDisabled ||
+                                                    ADVANCED_ACTION_GROUP_IDS.includes(
+                                                        //@ts-ignore
+                                                        item.id
+                                                    ) ||
+                                                    ADVANCED_ACTION_IDS.includes(
+                                                        //@ts-ignore
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionListPanel/ActionList/ActionList.tsx]: create supertype for all actions types as union 

avoid ts-ignore

**Review by preludium — DISMISSED** [review]

File: runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionListPanel/ActionList/ActionList.tsx [hunk 2]
```diff
@@ -111,19 +110,18 @@ const ActionList: FC<ActionListProps> = ({
                                 >
                                     <List component="div" disablePadding>
                                         {items.map((item: Item) => {
+
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionListPanel/ActionList/ActionList.tsx:114]: remove this empty line

File: runbotics/runbotics-common/src/actions/actions.ts [hunk 3]
```diff
@@ -152,3 +152,24 @@ export enum ACTIONS_GROUPS {
     CLOSE = 'application.close',
   }
 
+  export type AllActions =
```
Comment (star-in-the-jar) [inline runbotics/runbotics-common/src/actions/actions.ts]: enum name should be always singular -> rename to AllAction

Comment (star-in-the-jar) [reply runbotics/runbotics-common/src/actions/actions.ts]: or just Action

or event ActionId if this is about id

**Review by star-in-the-jar — CHANGES_REQUESTED** [review]

File: runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionListPanel/ActionListPanel.types.ts [hunk 4]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionListPanel/ActionListPanel.types.ts:1]: NOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO

File: runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionListPanel/ActionList/ActionList.tsx (hunk 2 above)
Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionListPanel/ActionList/ActionList.tsx:114]: READ COMMENTS

File: runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionListPanel/ActionList/ActionList.tsx [hunk 5]
```diff
@@ -82,10 +82,10 @@ const ActionList: FC<ActionListProps> = ({
                 }
             >
                 {groups.map(({ key, label, items }) => {
+
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionListPanel/ActionList/ActionList.tsx:85]: NOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO V2

**Review by star-in-the-jar — APPROVED** [review]

**Pull request merged**